src/pages/user/Settings.js

```diff
@@ -106,6 +125,22 @@ export default class UserSettings extends Component {
               </InputGroup>
             </Col>
           </Row>
+          <Row></Row>
```

Comment:
Why is this empty Row here?

Reply:
I wanted to add some space in between the togglebutton and the content above, but there's probably better ways to do that.